internal/generator/vector/normalize/schema/otel/transform.go

```diff
@@ -0,0 +1,58 @@
+package otel
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/openshift/cluster-logging-operator/internal/generator"
+	. "github.com/openshift/cluster-logging-operator/internal/generator/vector/elements"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+)
+
+func ID(id1, id2 string) string {
+	return fmt.Sprintf("%s_%s", id1, id2)
+}
+
+func Transform(id string, inputs []string) Element {
+	return Remap{
+		Desc:        "Normalize log records to OTEL schema",
+		ComponentID: id,
+		Inputs:      helpers.MakeInputs(inputs...),
+		VRL: strings.TrimSpace(`
+# Tech preview, OTEL for application logs only
+if .log_type == "application" {
+	# Convert @timestamp to nano and delete @timestamp
+	.timeUnixNano = to_unix_timestamp(to_timestamp!(del(.@timestamp)))
+
+	.severityText = del(.level)
+
+	# Convert syslog severity keyword to number, default to 9 (unknown)
+	.severityNumber = to_syslog_facility(.severityText) ?? 9
```

Comment:
severity is not the same as facility (e.g. https://success.trendmicro.com/dcx/s/solution/TP000086250?language=en_US) .  Regardless, I think our fluent code has 0-9 but I believe 0-7 have always been the only ones available